gax/src/main/java/com/google/api/gax/rpc/ApiClientHeaderProvider.java

```diff
@@ -172,6 +179,15 @@ public Builder setTransportToken(String name, String version) {
       return this;
     }
 
+    public String getQuotaProjectIdToken() {
```

Comment:
Can you add docs explaining what a quota project is and when I would want to set it?